Wireframe/css/style.css

```diff
@@ -0,0 +1,52 @@
+body {
+  font-family: "Times New Roman", Times, serif;
+  font-size: 1rem;
+}
+
+main {
+  margin: 20px 250px;
+}
+
+img {
```

Comment:
consider that this means _all_ images will have this height and width. It is better to use classes for things that you don't want applied to everything or it can cause interesting bugs to crop up later in a project.